deep-learning/README.md

```diff
@@ -44,6 +45,7 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 
```

Comment:
Move this section to the NLP readme, under a new section called "More Courses" after the "Courses" section.

Reply:
Done!

Reply:
@lisabecker Better to do it in the same PR so that it's easier to review.